src/App.tsx

```diff
@@ -1,59 +1,58 @@
-import React from 'react';
+import * as React from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import './App.scss';
-import GameCanvas from './GameCanvas';
-import Banner from './Banner';
+
 import Menu from './Menu';
 import Button from 'react-bootstrap/Button';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faBars } from '@fortawesome/free-solid-svg-icons'
-import ConnectionPrompt from './ConnectionPrompt';
-import { UserManager, UserStateType, IUserListener } from './model/UserManager';
+import Banner from './Banner';
+import FourPlay from './pages/fourplay/FourPlay';
+import { NotFound } from './pages/notfound/NotFound';
+import { Support } from './pages/support/Support';
+
+interface IAppProps {
+}
 
-interface IAppProps {}
 interface IAppState {
     modalShow: boolean;
-    message: JSX.Element;
 }
 
-export default class App extends React.Component<IAppProps, IAppState> implements IUserListener {
-    private userManager: UserManager = new UserManager();
-
+export default class App extends React.Component<IAppProps, IAppState> {
     constructor(props: IAppProps) {
         super(props);
         this.state = {
-            message: <span></span>,
             modalShow: false,
         }
-
-        this.userManager.addListener(this);
     }
 
     public render() {
-         return (
+        return (
+           <BrowserRouter>
             <div className="App">
-            <Banner/>
-            <div id="alert">
-                {this.state.message}
-            </div>
-            <GameCanvas userManager={this.userManager} setMessage={message => this.setState({message})}/>
-
-            {this.userManager.getUserState() !== UserStateType.Connected &&
-                <ConnectionPrompt userManager={this.userManager}/>
-            }
-
-            <Button
-                variant="primary"
-                id="menu-button"
-                onClick={() => this.setState({modalShow: true})}
-            >
-                <FontAwesomeIcon icon={faBars} />
-            </Button>
-
-            <Menu
-                show={this.state.modalShow}
-                onHide={() => this.setState({modalShow: false})}
-            />
+                <Banner/>
+                <div className="main-content">
+                <Switch>
+                    <Route path="/" component={FourPlay} exact/>
```

Comment:
Will this be ok as we keep adding games to the list? Like when it becomes as long as 10 games or 50 games?

Reply:
Maybe, it's not going to load all the games at once. Just going to be a lot of games in that one file. We can probably figure out a solution to it when it becomes a problem.

Reply:
I meant more like should we have an array of route objects and render those in a for loop?